web/geo/generateAggregates.ts

```diff
@@ -29,31 +42,30 @@ const generateAggregates = (geojson, zones: ZoneConfig) => {
   const combinedZones = countryZonesToCombine
     .map((country) => {
       if (country === undefined) {
-        return null;
+        return emptyFeature;
```

Comment:
Because if we change the FeatureCollection type to allow `features: Feature | null` we would have to add null-checks all over the code. And since we actually never want a feature to be `null`, I thought this seemed like the best option - although I also don't think it's great :)
Do you have a better idea?

Reply:
Maybe we can filter the list first?

Somewhat related stackoverflow link: https://stackoverflow.com/questions/24806772/how-to-skip-over-an-element-in-map